etc/config/openclc.amazon.properties

```diff
@@ -117,6 +117,14 @@ compiler.armv7-oclcclang-trunk.semver=(trunk)
 # Arm v7-a with Neon and VFPv3
 compiler.armv7-oclcclang-trunk.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi --sysroot=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi/arm-unknown-linux-gnueabi/sysroot
 
+compiler.armv7-oclcclang-trunk-assertions.name=armv7-a clang (trunk, assertions)
+compiler.armv7-oclcclang-trunk-assertions.exe=/opt/compiler-explorer/clang-assertions-trunk/bin/clang
+compiler.armv7-oclcclang-trunk-assertions.demangler=/opt/compiler-explorer/gcc-snapshot/bin/c++filt
+compiler.armv7-oclcclang-trunk-assertions.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
+compiler.armv7-oclcclang-trunk-assertions.semver=(assertions trunk)
+# Arm v7-a with Neon and VFPv3
+compiler.armv7-oclcclang-trunk.options=-target arm-linux-gnueabi --gcc-toolchain=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi --sysroot=/opt/compiler-explorer/arm/gcc-8.2.0/arm-unknown-linux-gnueabi/arm-unknown-linux-gnueabi/sysroot
```

Comment:
is this missing the `-assertions` suffix on the compiler name?

Reply:
Thanks! That's what I get for copy pasting without double checking properly